src/files/header/Header.js

```diff
@@ -10,18 +10,12 @@ import Button from '../../components/button/Button'
 // Icons
 import GlyphDots from '../../icons/GlyphDots'
 
-function BarOption ({ children, title, isLink = false, className = '', ...etc }) {
-  className += ' tc pa3'
-
-  if (etc.onClick) className += ' pointer'
-
-  return (
-    <div className={className} {...etc}>
-      <span className='nowrap db f4 charcoal'>{children}</span>
-      <span className={`db ttu f6 montserrat fw4 ${isLink ? 'link' : 'charcoal-muted'}`}>{title}</span>
-    </div>
-  )
-}
+const BarOption = ({ children, title, isLink = false, className = '', ...etc }) => (
+  <div className={classNames(className, 'tc pa3', etc.onClick && 'pointer')} {...etc}>
+    <span className='nowrap db f4 charcoal'>{children}</span>
+    <span className={`db ttu f6 montserrat fw4 ${isLink ? 'link' : 'charcoal-muted'}`}>{title}</span>
```

Comment:
```suggestion
    <span className={`nowrap db ttu f6 montserrat fw4 ${isLink ? 'link' : 'charcoal-muted'}`}>{title}</span>
```

This should take care of the visual wierdness mentioned in https://github.com/ipfs-shipyard/ipfs-webui/pull/1685#discussion_r517561452 ?

Reply:
Oh sorry, missed that one! Good catch 👍